docs/redoc/src/main/scala/sttp/tapir/redoc/Redoc.scala

```diff
@@ -29,8 +35,8 @@ object Redoc {
     |  </style>
     |</head>
     |<body>
-    |<redoc spec-url='$specUrl' expand-responses="200,201"></redoc>
-    |<script src="https://cdn.jsdelivr.net/npm/redoc@$redocVersion/bundles/redoc.standalone.js"></script>
+    |  <redoc spec-url='$specUrl' expand-responses="200,201" $redocOptions theme='$redocThemeOptionsJson'></redoc>
```

Comment:
will an empty value work fine here? or should we omit `theme` entirely when `redocThemeOptionsJson == ""` ?

Reply:
Good point.

Double checked it, and yes, the value of `theme` attribute must be a valid JSON, otherwise documentation rendering breaks.

Do you think I should add simple JSON validation (and not just whether the string is empty)? Perhaps a regexp, since the `docs` module has very few dependencies, and there's no point in introducing a JSON dependency for this.